src/app/component/SearchField/SearchField.component.js

```diff
@@ -93,9 +93,11 @@ class SearchField extends PureComponent {
     handleChange = (e) => {
         const { target: { value } } = e;
         const { onSearchBarChange } = this.props;
-        onSearchBarChange(e);
+        const trimmedValue = value.trim();
+
+        onSearchBarChange({ target: { value: trimmedValue } });
```

Comment:
Wow that's very unsafe, do not fake events like this. Please change the `onSearchBarChange` to trim, rather then faking input.

Reply:
trimStart should be placed in search preview then